fmc/src/flow/rt_alias.rs

```diff
@@ -104,6 +106,11 @@ impl RtAliasLayer {
         Self::extend_pcrs(env, hand_off)?;
         cprintln!("[alias rt] Extend RT PCRs Done");
 
+        cprintln!("[alias rt] Lock RT PCRs");
+        env.pcr_bank.set_pcr_lock(caliptra_common::FMC_CURRENT_PCR);
```

Comment:
The comment says we're locking RT PCRs, but the ID is FMC's PCR.

Reply:
renamed it.